chroma-manager/chroma_core/migrations/0037_remove_rsyslog_types.py

```diff
@@ -42,1497 +42,1496 @@ def forwards(self, orm):
         map(delete_item, content_types)
 
         # Remove any entries from chroma_core_job.locks_json that have a locked_item_type_id referencing one of the rsyslog types
-    	jobs = orm['chroma_core.Job'].objects.all()
+        jobs = orm['chroma_core.Job'].objects.all()
         map(partial(update_locks_json, content_type_ids), jobs)
 
         # Remove any entries from chroma_core_job.wait_for_json whose entry points to a job id that no longer exists. These entries
         # were removed when the content type id was deleted.
         jobs = orm['chroma_core.Job'].objects.all()
         job_ids = map(get_id, jobs)
-        map(partial(update_wait_for_json_entries, job_ids) ,jobs)
+        map(partial(update_wait_for_json_entries, job_ids), jobs)
 
 
     def backwards(self, orm):
-        "Write your backwards methods here."
 
-    models = {
```

Comment:
models should not be indented as if it's part of "backward", you need a "pass" for the backwards method (or just remove it as I think it's in the base class)

Reply:
I just removed the method since it's not being used.